aries-site/src/examples/components/layer/ConfigurationForm.js

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable react/prop-types */
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  DataTable,
+  Form,
+  FormField,
+  Heading,
+  Page,
+  PageContent,
+  TextInput,
+  CheckBoxGroup,
+  AnnounceContext,
+} from 'grommet';
+import {
+  useFilters,
+  FiltersProvider,
+  FilterControls,
+} from '../../templates/FilterControls';
+import applications from '../../../data/mockData/applications.json';
+import {
+  ConfirmationContext,
+  ConfirmationProvider,
+  DoubleConfirmation,
+  LayerHeader,
+  Sidedrawer,
+  useConfirmation,
+} from './components';
+
+const defaultFormValues = {
+  'application-title': '',
+  'publisher-title': '',
+  'pricing-select': [],
+  'delivery-select': [],
+};
+
+export const ConfigurationForm = () => (
+  <ConfirmationProvider>
+    <ConfirmationContext.Consumer>
+      {({ showLayer, showConfirmation }) => (
+        <>
+          <ApplicationsPage />
+          {showLayer ? <AddApplication /> : null}
```

Comment:
Should this layer get a container ref as well?

Reply:
I was a running into a weird scroll issue when I set a target on the layer -- maybe tomorrow we can connect to see if you have any thoughts? I wasn't able to figure it out.

otherwise, thinking we can file it as next steps work so it's not a blocker to publishing the guidance.

Reply:
Filed issue here: https://github.com/grommet/grommet/issues/6532